packages/salesforcedx-vscode-apex/src/helpers/languageServerUtils.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {
+  extensionUris,
+  projectPaths
+} from '@salesforce/salesforcedx-utils-vscode';
+import { copyFileSync, existsSync, unlinkSync } from 'fs';
+import { join } from 'path';
+import * as vscode from 'vscode';
+import { VSCODE_APEX_EXTENSION_NAME } from '../constants';
+
+const setupDB = (): void => {
+  if (
+    vscode.workspace.workspaceFolders &&
+    vscode.workspace.workspaceFolders[0]
+  ) {
+    const dbPath = projectPaths.apexLanguageServerDatabase();
+    if (existsSync(dbPath)) {
+      unlinkSync(dbPath);
+    }
+
+    try {
```

Comment:
Same story as the icons logic wise.  This was assuming previously it's location in the file system to find the resources directory ala `const systemDb = path.join(__dirname, '..', '..', 'resources', 'apex.db');`
I updated it to use the extensionUri method to find the extension directory which works for both dev and published versions.

Reply:
Solid